server/src/services/auth.ts

```diff
@@ -32,6 +32,16 @@ const signupUser = async ({
   return userInfo.id;
 };
 
+const checkAdmin = async ({ username }) => {
+  const getAdmin:any = await Admin.findOne({
```

Comment:
```suggestion
  const getAdmin:any = await Admin.findOne({ 
```
remove any from here, and it's not a function to have an action name